scene/gui/tab_bar.cpp

```diff
@@ -998,7 +1021,7 @@ void TabBar::_update_cache(bool p_update_hover) {
 		tabs.write[i].size_text = Math::ceil(tabs[i].text_buf->get_size().x);
 		tabs.write[i].size_cache = get_tab_width(i);
 
-		if (max_width > 0 && tabs[i].size_cache > max_width) {
+		if ((tabs.write[i].truncated = max_width > 0 && tabs[i].size_cache > max_width)) {
```

Comment:
Is this a typo? If not it should be:
```suggestion
		tabs.write[i].truncated = max_width > 0;
		if (max_width > 0 && tabs[i].size_cache > max_width) {
```

Reply:
Yes sorry, the compiler on my computer didn't give any errors :(

Reply:
So should it be?
```suggestion
		if (tabs.write[i].truncated == max_width > 0 && tabs[i].size_cache > max_width) {
```
What is the intention of this code?

Reply:
> So should it be?

I meant the equivalent of this code here:
```C++
tabs.write[i].truncated = max_width > 0 && tabs[i].size_cache > max_width;
if (tabs[i].truncated) {
}
```